src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -25,18 +28,33 @@ public Joystick getOperatorJoystick() {
 	    }
 	    
 	    public OI() {
-
+	    	this(new Joystick(0), new Joystick(1));
+	    }
+	    
+	    public OI(Joystick driverStick, Joystick controlStick)  {
 			// Joystick for driving the robot around
-			driverStick = new Joystick(0);
+			this.driverStick = driverStick;
+
+			// Joystick for controlling the mechanisms of the robot
+			this.controlStick = controlStick;
 			
-			EJoystickButton LB = new EJoystickButton(driverStick, 5);	
+			EJoystickButton LB = new EJoystickButton(controlStick, 5);	
 			// TODO: Tune stopCycles and speed
 			QuickStraightDriveWithDistanceSensor sdwe = new QuickStraightDriveWithDistanceSensor(0.5, 13.0, 15.0, 4, 0.8);
```

Comment:
I'm pretty sure I did, but I can try it again once I fix all the things from this review